assets/css/buttons.css

```diff
@@ -14,13 +14,13 @@
   font-weight: 400;
   font-style: normal;
   line-height: 1;
-  font-size: .75rem;
+  font-size: .75em;
```

Comment:
Good point... It depends on whether we want every font/relevant elements to scale up at the same ratio. Some elements may not or should not scale up or down. I don't think we need to scale relatively but if that's our use case we should use em. Otherwise, px.